website/docs/configuration/recommended-rbac-configuration.mdx

```diff
@@ -178,3 +180,142 @@ roleRef:
   apiGroup: rbac.authorization.k8s.io
 ---
 ```
+
+
+## Weave GitOps Enterprise RBAC considerations  <TierLabel tiers="enterprise" />
+
+Weave GitOps Enterprise extends Weave GitOps permissions by adding more roles. These roles may need to be extended further in order to support certain use cases. Some of the most common use cases are described below.
+
+### Viewing Flagger-generated resources
```

Comment:
Given we've pitched these as "use cases" - suggest rewording as "Progressive delivery with Flagger"

Reply:
Sure 👍